wiki/content/deploy/index.md

```diff
@@ -1706,15 +1739,16 @@ If the `--tls_client_auth` option is set to `REQUEST`or `VERIFYIFGIVEN` (default
 use the option `--cacert`. For instance (for an export request):
 
 ```
-curl --cacert ./tls/ca.crt https://localhost:8080/admin/export
+curl --silent --cacert ./tls/ca.crt https://localhost:8080/admin/export
```

Comment:
why --silent?

Reply:
I was getting progress bar for a small query.   Do we want users to have progress bar for small queries.  Progress bars are fine for downloading tarballs, but for API, not the preferred experience.

Reply:
ok.